mesondev.sh

```diff
@@ -27,15 +27,16 @@ python -c "from scipy import constants as s; s.test()"
 # python -c "from scipy import ndimage as s; s.test()"  # relies on special
 python -c "from scipy import odr as s; s.test()"
 python -c "from scipy.sparse import csgraph as s; s.test()"
-# python -c "from scipy import sparse as s; s.test()"  # relies on sparse.linalg
+python -c "from scipy.sparse import linalg as s; s.test()"
```

Comment:
There was one random test failure, I sent a PR to the main SciPy repo to fix the problem. The tests pass then.